apps/omg_api/config/config.exs

```diff
@@ -4,7 +4,9 @@ config :omg_api,
   deposit_finality_margin: 10,
   exiters_finality_margin: 11,
   submission_finality_margin: 20,
-  ethereum_status_check_interval_ms: 6_000,
+  ethereum_events_check_interval_ms: 500,
```

Comment:
I have split the responsibilities of this interval setting. It used to be related to new eth-blocks being acknowledged. Now it only determines the cadence of `EthereumEventsListener` waking up, which is good to have quite frequent in the new approach to syncing (the forced `:sync` message became a bit obsolete in the new approach).

If we devise some upgraded politics to force a `:sync` message, we might go back to the large interval. But in the "new approach" it also isn't necessary to speed up by sending these `:sync` messages all the time (because we consume ethereum events in large chunks, as opposed to a single eth_height at a time).

Hope this clarifies a bit